src/test/java/com/znsio/sample/e2e/steps/AmazonCartFunctionalitySteps.java

```diff
@@ -1,2 +1,62 @@
-package com.znsio.sample.e2e.steps;public class AmazonCartFunctionalitySteps {
+package com.znsio.sample.e2e.steps;
+
+import com.context.SessionContext;
+import com.context.TestExecutionContext;
+import com.znsio.e2e.runner.Runner;
+import com.znsio.e2e.tools.Drivers;
+import com.znsio.sample.e2e.businessLayer.Amazon.AmazonHomeBL;
+import com.znsio.sample.e2e.businessLayer.Amazon.ProductCartDetailsBL;
+import com.znsio.sample.e2e.businessLayer.Amazon.ProductDetailsBL;
+import com.znsio.sample.e2e.businessLayer.Amazon.SearchResultsBL;
+import com.znsio.sample.e2e.businessLayer.notepad.NotepadBL;
+import com.znsio.sample.e2e.entities.SAMPLE_TEST_CONTEXT;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.apache.log4j.Logger;
+
+public class AmazonCartFunctionalitySteps {
```

Comment:
Class name does not need to say Functionality. Just saying AmazonCartSteps is sufficient